styles/go.dev/catppuccin.user.css

```diff
@@ -0,0 +1,930 @@
+        content: url('data:image/svg+xml,@{svg}');
+        filter: none;
+      }
+
+      .go-Carousel-prevSlide img {
+        @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{text}"><path d="m14 7-5 5 5 5z"/><path d="M24 0v24H0V0z" fill="none" opacity=".87"/></svg>');
+        content: url('data:image/svg+xml,@{svg}');
+        filter: none;
+      }
+    }
+    // Alert Chips
+    .go-Chip--alert {
+      color: @crust;
+    }
+    // Pkg.go Clipboard Icon
+    .go-Clipboard img.go-Icon {
+      @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{accent-color}"><path d="M0 0h24v24H0z" fill="none"/><path d="M18 21H4V7H2v14c0 1.1.9 2 2 2h14zm3-4V3c0-1.1-.9-2-2-2H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2m-2 0H8V3h11z"/></svg>');
+      content: url('data:image/svg+xml,@{svg}');
+      filter: none;
+    }
+    // Pkg.go Footer
+    .go-Footer {
+      img[alt='Dark theme'] {
+        @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{text}"><path d="M0 0h24v24H0z" fill="none"/><path d="M10 2c-1.82 0-3.53.5-5 1.35C7.99 5.08 10 8.3 10 12s-2.01 6.92-5 8.65C6.47 21.5 8.18 22 10 22c5.52 0 10-4.48 10-10S15.52 2 10 2"/></svg>');
+        content: url('data:image/svg+xml,@{svg}');
+        filter: none;
+      }
+
+      img[alt='Light theme'] {
+        @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" height="24px" viewBox="0 0 24 24" width="24px" fill="@{text}"><rect fill="none" height="24" width="24"/><path d="M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5M2 13h2c.55 0 1-.45 1-1s-.45-1-1-1H2c-.55 0-1 .45-1 1s.45 1 1 1m18 0h2c.55 0 1-.45 1-1s-.45-1-1-1h-2c-.55 0-1 .45-1 1s.45 1 1 1M11 2v2c0 .55.45 1 1 1s1-.45 1-1V2c0-.55-.45-1-1-1s-1 .45-1 1m0 18v2c0 .55.45 1 1 1s1-.45 1-1v-2c0-.55-.45-1-1-1s-1 .45-1 1M5.99 4.58a.996.996 0 0 0-1.41 0 .996.996 0 0 0 0 1.41l1.06 1.06c.39.39 1.03.39 1.41 0s.39-1.03 0-1.41zm12.37 12.37a.996.996 0 0 0-1.41 0 .996.996 0 0 0 0 1.41l1.06 1.06c.39.39 1.03.39 1.41 0a.996.996 0 0 0 0-1.41zm1.06-10.96a.996.996 0 0 0 0-1.41.996.996 0 0 0-1.41 0l-1.06 1.06c-.39.39-.39 1.03 0 1.41s1.03.39 1.41 0zM7.05 18.36a.996.996 0 0 0 0-1.41.996.996 0 0 0-1.41 0l-1.06 1.06c-.39.39-.39 1.03 0 1.41s1.03.39 1.41 0z"/></svg>');
+        content: url('data:image/svg+xml,@{svg}');
+        filter: none;
+      }
+
+      img[alt='System theme'] {
+        @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{text}"><path d="M0 0h24v24H0z" fill="none"/><path d="M20 15.31 23.31 12 20 8.69V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20zM12 18V6c3.31 0 6 2.69 6 6s-2.69 6-6 6"/></svg>');
+        content: url('data:image/svg+xml,@{svg}');
+        filter: none;
+      }
```

Comment:
Are these dark/light/system icons the same as on `go.dev`? Let's try to avoid duplicating these as well.